src/Services/privateApiService.js

```diff
@@ -6,14 +6,24 @@ const config = {
     }
 }
 
+const privateInstance = axios.create({
```

Comment:
axiosInstance maybe its a better name. Then if you have a privateInstance of AWS you can call it awsInstance, and so on. privateInstance its to generic. Also, its call private instance and the baseURl says public.

Reply:
Done